tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@tsconfig/node18/tsconfig.json",
+  "compilerOptions": {
```

Comment:
we'll have to revisit this file when we are building/packaging this later, this is settings for esbuild-lambda!